Sources/Events/Networking/EventsRequest.swift

```diff
@@ -32,10 +32,18 @@ struct EventsRequest {
                 }
                 return AnyEncodable(event)
             case .customerCenter:
-                guard let event = CustomerCenterEvent(storedEvent: storedEvent) else {
-                    return nil
+                switch storedEvent.eventDiscriminator {
+                case "survey_option_chosen":
```

Comment:
Hmm can we store these into some sort of enum? Would be better to avoid hardcoded strings

Reply:
oh I need to do this